executor/show_test.go

```diff
@@ -970,6 +970,34 @@ func (s *testSuite5) TestShowCreateTable(c *C) {
 			"  PRIMARY KEY (`id`) /*T![clustered_index] CLUSTERED */,\n"+
 			"  KEY `idx_v` (`v`) COMMENT 'foo''bar'\n"+
 			") ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin"))
+
+
+	// For issue #29922
+	tk.MustExec("CREATE TABLE `thash` (\n  `id` bigint unsigned NOT NULL,\n  `data` varchar(255) DEFAULT NULL,\n  PRIMARY KEY (`id`)\n)\nPARTITION BY HASH (`id`)\n(PARTITION pEven COMMENT = \"Even ids\",\n PARTITION pOdd COMMENT = \"Odd ids\");")
```

Comment:
I think there should also be a test for '... PARTITION BY HASH (id) PARTITIONS 2' as well as one with named partitions, but with default names like '... PARTITION BY HASH (id) (PARTITION p0, PARTITION p1)', what do you think?

Reply:
yes, but it already exists in an old case in `executor/seqtest/seq_executor_serial_test.go`, and I fixed it.